python/hail/dataset.py

```diff
@@ -750,6 +750,41 @@ def annotate_variants_expr(self, condition):
         pargs = ['annotatevariants', 'expr', '-c', condition]
         return self.hc.run_command(self, pargs)
 
+    def annotate_variants_keytable(self, keytable, condition):
```

Comment:
I'd add a call to this in tests.py just to make sure the plumbing works.  You don't even need to check the correctness since you test in Scala.